app/pulseapi/entries/admin.py

```diff
@@ -31,16 +46,27 @@ class EntryAdmin(admin.ModelAdmin):
 
     readonly_fields = (
         'id',
+        'created',
         'thumbnail_url',
         'thumbnail_image_tag',
         'bookmark_count',
     )
 
+    ordering = (
+        'created',
```

Comment:
do we want `-created` so that the newest entry is the first one, rather than the last one?